### ISSUE TITLE
[1.9.29] iPhone 6 Plus / 10.0.2 - Black screen is displayed at preview.

### ISSUE THREAD

**Issue by testershub** [Lomotif/lomotif-ios-issues#825]
Frequency: Every time

Action:
1: Launch the app.
2: Tap on "Start" and select 1 photo from selfie folder.
3: Add music and proceed to "Create" screen.
4: Change the view to rectangle (16:9).
5: Tap on "Export" option. (User will navigated to "Share" screen)
6: Tap on back option "<".
7: Tap on "+" option and add recently created lomotif.
8: Check if the black screen is displayed at preview.

Expected result: Preview should be displayed without black frame.

Actual result: Black screen is displayed at preview when user add recently created lomotif from "Create" screen.

**Comment by hiteshbheda** [comment]
@xiuie I can't reproduce this issue for build [1.9.29] on iPhone 6 / iOS 10.0.2

**Comment by testershub** [comment]
@hiteshbheda Can you try to reproduce this issue by taking image from "Selfie" folder?

**Comment by hiteshbheda** [comment]
@xiuie @testershub No, still I can't reproduce this issue even with "Selfie" folder image on iPhone 6 / iOS 10.0.2

**Comment by vysohanych** [comment]
@xiuie I can't reproduce this issue on build 1.9.29 on iPad 2 / iOS 9.3.5

**Comment by Dipamit** [comment]
@xiuie : Not able to reproduce this issue on iPhone 5/9.3.4 for build 1.9.29

**Comment by vysohanych** [comment]
@xiuie I can't reproduce this issue on build 1.9.35 on iPhone 5S / iOS 9.3.5

**Comment by hiteshbheda** [comment]
@xiuie I can't reproduce this issue for build [1.9.37] on iPhone 6 / iOS 10.0.2

**Comment by testershub** [comment]
@xiuie I can still randomly reproduce this issue for build [1.9.37] on iPhone 6 Plus / 10.0.2.
